.github/workflows/buildsCI.yaml

```diff
@@ -400,4 +400,46 @@ jobs:
        cmake --build . --config ${{env.LRS_BUILD_CONFIG}} -- -j4
        ls
   
-  
+  cppcheck:
+    name: cppcheck-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        
+      - name: Check_API
+        shell: bash
+        run: |
+          cd scripts
+          ./api_check.sh
+          cd ..  
+        
+      - name: Prerun
+        shell: bash
+        run: |
+          sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test;
+          sudo apt-get update;
+          sudo apt-get install -qq cppcheck;
+      
+      - name: Run
+        shell: bash
+        run: | 
+          cppcheck . --std=c++11 -ithird-party/easyloggingpp -ithird-party/glad -ithird-party/sqlite -ithird-party/imgui -ithird-party/glfw -ithird-party/hidapi -ithird-party/json.hpp --xml &> cppcheck_run.log
+
+      - name: Upload cppcheck log
+        uses: actions/upload-artifact@v3
+        with: 
+          name: cppcheck_log
+          path: cppcheck_run.log
+
+      - name: Provide correct exit status for job 
+        shell: bash
+        run: |
+          ERROR_COUNT=$(grep cppcheck_run.log -e "severity=\"error\"" -c);
+          if [ $ERROR_COUNT == 53 ];
```

Comment:
Please add a comment before.
"Once this is reduced, we need to reduce the error count number as well"

Reply:
I even say place a global variable that set the count and use it here.
And only this variable will be updated on future changes.
Under env like [here](https://github.com/IntelRealSense/librealsense/blob/923253f645e867c75e3c6c30a84127f41d82f3e9/.github/workflows/buildsCI.yaml#L9)?

Reply:
Created environment variable, added commment